concrete-core-fixture/src/fixture/glwe_ciphertext_trivial_encryption.rs

```diff
@@ -0,0 +1,134 @@
+    type SamplePrototypes =
+        (<Maker as PrototypesPlaintextVector<Precision>>::PlaintextVectorProto,);
+    type PreExecutionContext = (PlaintextVector,);
+    type PostExecutionContext = (PlaintextVector, Ciphertext);
+    type Criteria = (Variance,);
+    type Outcome = (Vec<Precision::Raw>, Vec<Precision::Raw>);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![
+                GlweCiphertextTrivialEncryptionParameters {
+                    glwe_dimension: GlweDimension(200),
+                    polynomial_size: PolynomialSize(256),
+                },
+                GlweCiphertextTrivialEncryptionParameters {
+                    glwe_dimension: GlweDimension(1),
+                    polynomial_size: PolynomialSize(256),
+                },
+            ]
+            .into_iter(),
+        )
+    }
+
+    fn generate_random_repetition_prototypes(
+        _parameters: &Self::Parameters,
+        _maker: &mut Maker,
+    ) -> Self::RepetitionPrototypes {
+    }
+
+    fn generate_random_sample_prototypes(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+        _repetition_proto: &Self::RepetitionPrototypes,
+    ) -> Self::SamplePrototypes {
+        let raw_plaintext_vector = Precision::Raw::uniform_vec(parameters.polynomial_size.0);
+        let proto_plaintext_vector =
+            maker.transform_raw_vec_to_plaintext_vector(raw_plaintext_vector.as_slice());
+        (proto_plaintext_vector,)
+    }
+
+    fn prepare_context(
+        _parameters: &Self::Parameters,
+        maker: &mut Maker,
+        _repetition_proto: &Self::RepetitionPrototypes,
+        sample_proto: &Self::SamplePrototypes,
+    ) -> Self::PreExecutionContext {
+        let (proto_plaintext_vector,) = sample_proto;
+        (maker.synthesize_plaintext_vector(proto_plaintext_vector),)
+    }
+
+    fn execute_engine(
+        parameters: &Self::Parameters,
+        engine: &mut Engine,
+        context: Self::PreExecutionContext,
+    ) -> Self::PostExecutionContext {
+        let (plaintext_vector,) = context;
+        let ciphertext = unsafe {
+            engine.trivially_encrypt_glwe_ciphertext_unchecked(
+                parameters.glwe_dimension.to_glwe_size(),
+                &plaintext_vector,
+            )
+        };
+        (plaintext_vector, ciphertext)
+    }
+
+    fn process_context(
+        _parameters: &Self::Parameters,
+        maker: &mut Maker,
+        _repetition_proto: &Self::RepetitionPrototypes,
+        sample_proto: &Self::SamplePrototypes,
+        context: Self::PostExecutionContext,
+    ) -> Self::Outcome {
+        let (proto_plaintext_vector,) = sample_proto;
+        let (plaintext_vector, ciphertext) = context;
+        let proto_output_ciphertext = maker.unsynthesize_glwe_ciphertext(&ciphertext);
+        let proto_output_plaintext_vector =
+            maker.trivial_decrypt_glwe_ciphertext(&proto_output_ciphertext);
```

Comment:
Here it should be trivially_decrypt instead of trivial_decrypt to match the wording in other functions.